dcos/config.py

```diff
@@ -55,9 +55,13 @@ def get_config(mutable=False):
     return load_from_path(path, mutable)
 
 
-def get_config_val(name, config=None):
-    """Returns the config value for the specified key. Looks for corresponding
-    environment variable first, and if it doesn't exist, uses the config value.
+def get_config_val_and_envvar(name, config=None):
+    """Returns a tuple of the config value for the specified key and
```

Comment:
both get_config_val and get_config_val_and_envvar are returning the same value, which is the effective_value in the overwrite case (the one from the environment variable).  Neither of these return the original value in the case that the environment variable triggers an overwrite.  

(sorry, wasn't sure entirely sure what "the overwritten value" meant.)